CONTRIBUTING.md

```diff
@@ -39,7 +39,7 @@ What to work on
 
 Good starting projects are listed on [Bugzilla][bugzilla]. If you have any questions, just ask in the #mdndev IRC channel on irc.mozilla.org!
```

Comment:
I propose we expand the copy and links:

We maintain [a list of mentored bugs](https://bugzilla.mozilla.org/buglist.cgi?f1=bug_mentor&list_id=11858776&o1=isnotempty&resolution=---&query_format=advanced&product=Mozilla%20Developer%20Network) that are good for first-time contributors.

We also tag [some bugs with "good first bug"](https://bugzilla.mozilla.org/buglist.cgi?j_top=OR&list_id=11858643&o1=isnotempty&o2=substring&v2=%5Bgood%20first%20bug%5D&f1=bug_mentor&columnlist=short_desc%2Ccomponent%2Cchangeddate&query_format=advanced&f2=status_whiteboard&bug_status=UNCONFIRMED&bug_status=NEW&bug_status=ASSIGNED&bug_status=REOPENED&product=Mozilla%20Developer%20Network) which may or may not have mentors.

Finally, [there are 'patchwelcome' bugs](https://bugzilla.mozilla.org/buglist.cgi?list_id=11858815&resolution=---&status_whiteboard_type=allwordssubstr&query_format=advanced&status_whiteboard=patchwelcome&product=Mozilla%20Developer%20Network) that may be good for more advanced contributors.